src/components/dropdown/dropdown.ts

```diff
@@ -506,15 +548,14 @@ class BXDropdown extends HostListenerMixin(FocusMixin(LitElement)) {
         <div
           role="button"
```

Comment:
Just want to verify that `role="button"` is valid if the trigger isn't focussable ...

Reply:
Good point! Fixed.